index.html

```diff
@@ -54,53 +62,61 @@ <h2 class="work-header">Tonic</h2>
             <div class="card-info">
             <h2 class="work-header">Multi-Post
                 Stories</h2>
             <ul class="descr-list">
-                <li class="descr-list-item1">CANOPY</li>
-                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
+                <li class="descr-list-item1">FACEBOOK</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png">Full Stack Dev</li>
                 <li class="descr-list-item"><img class="counter-img"src="images/Counter.png">2015</li>
             </ul>
-            <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p class="lang-descr">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
             <ul class="lang-list">
                 <li class="lang-list-descr">html</li>
+                <li class="lang-list-descr">Ruby on rails</li>
                 <li  class="lang-list-descr">css</li>
                 <li  class="lang-list-descr">javaScript</li>
             </ul>
             <button class="btn-project">See Project</button>
         </div>
+        <div class="section-img"><img src="images/snapshoot4.png"></div>
         </div>
-        <div class="card">
-            <div class="section-img"><img src="images/snapshoot3.png"></div>
+
+        <!-- ....card3 div.... -->
+        <div class="card card3">
+            <div class="section-img"><img src="images/snapshoot1.png"></div>
             <div class="card-info">
-            <h2 class="work-header">Tonic</h2>
+            <h2 class="work-header">Facebook 360</h2>
             <ul class="descr-list">
-                <li class="descr-list-item1">CANOPY</li>
-                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
+                <li class="descr-list-item1">FACEBOOK</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> Full Stack Dev</li>
                 <li class="descr-list-item"><img class="counter-img"src="images/Counter.png">2015</li>
             </ul>
-            <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p class="lang-descr">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
             <ul class="lang-list">
                 <li class="lang-list-descr">html</li>
+                <li class="lang-list-descr">Ruby on rails</li>
                 <li  class="lang-list-descr">css</li>
                 <li  class="lang-list-descr">javaScript</li>
             </ul>
-            <button class="btn-project">See Project</button>
+            <button class="btn-project btn-fab">See Project</button>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct style to all buttons as shown in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Uy7TAUKVJbGSY3Ol-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/213466821-45fbc50c-ae62-48b3-9c32-bf23a54bf3a3.png)